src/components/ScenarioParameters/components/ScenarioParametersInputs/ScenarioSelect.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import { Grid } from '@mui/material';
+import { Select } from '@cosmotech/ui';
+import { useSortedScenarioList } from '../../../../hooks/ScenarioListHooks';
+import { useCurrentScenarioId } from '../../../../state/hooks/ScenarioHooks';
+import { TranslationUtils } from '../../../../utils';
+
+export const ScenarioSelect = ({ parameterData, context, parameterValue, setParameterValue, isDirty }) => {
+  const { t } = useTranslation();
+  const scenarioList = useSortedScenarioList();
+  const currentScenarioId = useCurrentScenarioId();
+  const runTemplateWhiteList = parameterData.options?.runTemplateFilter;
```

Comment:
"whitelist" is a single word, I think there shouldn't be a capital "L" in the variable name
Also, maybe we could simply rename the variable to `runTemplateFilter` to stay consistent with the option name?